src/site/content/en/fast/remove-unused-code/index.md

```diff
@@ -24,6 +25,12 @@ download and use over _half a million_ public packages. But we often include
 libraries we're not fully utilizing. To fix this issue, **analyze your bundle**
 to detect unused code. Then remove **unused** and **unnecessary** libraries.
 
+By removing unused code, you can improve your website's [Core Web Vitals](/vitals/). [Largest Contentful Paint](/lcp/), for example, can be affected by unused code by increased bandwidth contention caused by large assets for [LCP candidates](/lcp/#what-elements-are-considered). LCP can also be affected if large JavaScript assets that render markup solely on the client [contain references to LCP candidates](/preload-scanner/#rendering-markup-with-client-side-javascript) by [delaying when these resources can load](/optimize-lcp/#1-eliminate-resource-load-delay).
```

Comment:
Suggest adding a heading.

Also even if the LCP isn't in the large bundle, it can still be slowed by contention so tweaked that slightly.

```suggestion
## Impact on Core Web Vitals

By removing unused code, you can improve your website's [Core Web Vitals](/vitals/). [Largest Contentful Paint](/lcp/), for example, can be affected by unused code by increased bandwidth contention caused by larger than necessary assets. LCP can also be affected if large JavaScript assets that render markup solely on the client [contain references to LCP candidates](/preload-scanner/#rendering-markup-with-client-side-javascript) by [delaying when these resources can load](/optimize-lcp/#1-eliminate-resource-load-delay).
```